### ISSUE TITLE
Survey Push Notifications Timezone Localization

### ISSUE THREAD

**Issue by hydawo** [onnela-lab/beiwe-backend#324]
Hassan to let Eli know when to put the code I've got on staging for testing.

**Comment by biblicabeebli** [comment]
updated to make it clear that you should tell me when to enable it on staging.

**Comment by biblicabeebli** [comment]
It was not deployed, I have now deployed it.
Just a reminder, while I have decent confidence in this code state, I'm not positive about it.

Oh, there's no frontend .... stuff.
I tried to build tests to cover it, but there may still be things like...... updating a survey shifts user timezones by their UTC offset forward/backward in time - or something, I'm making that up and I specifically had to spend time on "regenerating" correct push notification times, but this is the meme of the hardest thing to do right in software development. Timezones Suck.

**Comment by biblicabeebli** [comment]
And iOS doesn't report this yet.

**Comment by biblicabeebli** [comment]
Decision: remove study timezones. Its dangerous and complex.

**Comment by hydawo** [comment]
A user on iOS (TestFlight v2.4.6) and in a staging study ([Fucito Feature Testing 8.23.23](https://staging.beiwe.org/view_study/83)) reported she is receiving their study's surveys localized to her time in Ireland (she is on vacation). The study timezone is EST and the survey is scheduled to push at 9AM. She is receiving it at 9AM Ireland time.

Is this expected already?

**Comment by biblicabeebli** [comment]
Yes, only on Staging though.

**Comment by hydawo** [comment]
@biblicabeebli we had a real-world user from the Yale team testing on the Staging server study, [Fucito Feature Testing 8.23.23](https://staging.beiwe.org/view_study/83). The study is set up to deploy surveys daily at 9AM and the study timezone is set to EST.

They confirmed that while in Ireland the survey push notifications were arriving at 9AM local time!

Here are some additional questions I asked and Madi's responses:

What Beiwe ID are you in the staging study? I am zpg56mit, I think! 
- I can not remember how to see it in the app, and am not seeing it currently in there

When did you turn on airplane mode (I'm guessing prior to take off in Boston?) 
- yes prior to leaving

Did you (and if so when) turn airplane mode off when you arrive in Ireland? 
- I turned it off upon landing so we left Saturday night at 8:30 ish and arrived Sunday at 7:30 am or so.

Have you turned on your cell service or connected to a Wi-Fi point while there? If so, when? 
- I have had service the whole time Ireland is a supported country with T-Mobile, so I have had unlimited data and texting, other than calling. I turned my phone on immediately after landing, but it probably took an hour or so for it to actually start working.
